main.cpp

```diff
@@ -0,0 +1,419 @@
+#include <SFML/Graphics.hpp>
+#include <SFML/Network.hpp>
+#include <iostream>
+#include <windows.h>
+#include <map>
+#include "TextureHolder.h"
+#include "Square.h"
+#include "Piece.h"
+
+bool move(Square* slctd, Square* trgt, sf::UdpSocket& sckt, sf::IpAddress& IP, unsigned short& PORT, sf::Packet& pck);
+
+int main()
+{	
+	//kod debugowania = 1337
+	bool turn=false, gamestarted=false, iswhite = true;
+	sf::Packet pack;
+	sf::UdpSocket socket;
+	sf::IpAddress serveradress;
+	unsigned short serverport;
+	int serverconnectionstatus = 1, id, ctx, cty;
+	float tx, ty; //koordy targetu do update
```

Comment:
W jaki sposób różni się to od `ctx` i `cty`?
Z samej inicjalizacji tego nie widać, więc nazwy są złe.

Może jakaś klasa "Point"; wtedy `Point target;`